### PR TITLE
Footer update - Government Log in

### PR DESCRIPTION
**Pull request by claudielarouche**
Changing the footer link "Open Government Log In" 

from: 
http://registry.open.canada.ca/en/user/
to: 
http://registry.open.canada.ca/en/user/login

from:
http://registry.open.canada.ca/fr/user/
to: 
http://registry.open.canada.ca/fr/user/login

### CODE REVIEW

File: ckanext/canada/templates/public/footer.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
 <li><a href="{{ _('http://news.gc.ca/') }}">{{ _('News') }}</a></li>
 <li><a href="{{ _('https://www.canada.ca/en/government/system/laws.html') }}">{{ _('Treaties, laws and regulations') }}</a></li>
 <li><a href="{{ _('https://www.canada.ca/en/transparency/reporting.html') }}">{{ _('Government-wide reporting') }}</a></li>
-<li><a href="{{ _('/en/user') }}">{{ _('Open Government Log In') }}</a></li>
```
Comment (wardi) [inline ckanext/canada/templates/public/footer.html]: this template is for the public-facing site, we copy this template from the one Drupal uses which is /en/user/login

File: ckanext/canada/i18n/fr/LC_MESSAGES/canada.po [hunk 2]
```diff
@@ -1547,8 +1547,8 @@ msgstr "http://ouvert.canada.ca/fr/formulaire/faites-nous-part-de-vos-commentair
 msgid "Open Government Log In"
 msgstr "Ouverture de session Gouvernement ouvert"
 
-msgid "/en/user"
-msgstr "/fr/user"
```
Comment (wardi) [inline ckanext/canada/i18n/fr/LC_MESSAGES/canada.po]: this "translation" is weird, but we'll still need it for the public login urls

**Comment by claudielarouche** [comment]
Hi @wardi, I'm still learning how GitHub works, is there anything else I need to change for this pull request or did you modify everything that needed to be changed? (public facing footer)

Thanks!

**Comment by wardi** [comment]
No, I can't modify your fork so you'll have to make the changes I said above or close this and submit a new pull request.

**Comment by claudielarouche** [comment]
Hi @wardi,

I think I fixed my pull request now, does it look ok?

Thanks!

File: ckanext/canada/i18n/fr/LC_MESSAGES/canada.po (hunk 2 above)
Comment (wardi) [inline ckanext/canada/i18n/fr/LC_MESSAGES/canada.po]: we still need the /en/user "translation" for the places we use it on the public site

File: ckanext/canada/templates/public/ramp/ramp-en-ckan.html [hunk 3]
```diff
@@ -895,7 +895,7 @@ <h2 class="wb-inv">Site Information</h2>
         <h3>Contact information</h3>
         <ul class="list-unstyled">
           <li><a class="gl-footer" href="/en/forms/contact-us">Open Government Contact</a></li>
-          <li><a class="gl-footer" href="/en/user">Open Government Log In</a></li>
```
Comment (wardi) [inline ckanext/canada/templates/public/ramp/ramp-en-ckan.html]: this is on the public site, so doesn't need to be changed

File: ckanext/canada/templates/public/ramp/ramp-fr-ckan.html [hunk 4]
```diff
@@ -895,7 +895,7 @@ <h2 class="wb-inv">Information du site</h2>
       <h3>Coordonnées</h3>
       <ul class="list-unstyled">
         <li><a class="gl-footer" href="/fr/formulaire/faites-nous-part-de-vos-commentaires">Contactez Gouvernement ouvert</a></li>
-        <li><a class="gl-footer" href="/fr/user">Ouverture de session Gouvernement ouvert</a></li>
```
Comment (wardi) [inline ckanext/canada/templates/public/ramp/ramp-fr-ckan.html]: this is also on the public site, so doesn't need to be changed

**Comment by wardi** [comment]
@fanjinfei would you finish this PR off when you have time?

**Comment by fanjinfei** [comment]
no problem.

**Comment by fanjinfei** [comment]
new PR https://github.com/open-data/ckanext-canada/pull/880

**Comment by wardi** [comment]
done in #880

**Pull request closed without merging**